share/oneprovision/ansible/inventories/static_vxlan/group_vars/all

```diff
@@ -20,8 +20,9 @@ iptables_base_rules_strict: false
 
 # Parameters for static VXLAN connections between provisioned hosts
 opennebula_p2p_vxlan_bridge: vxbr100
-opennebula_p2p_vxlan_phydev: bond0:0
+opennebula_p2p_vxlan_phydev: "{{ ansible_interfaces | sort | select('in', ['bond0_0', 'eth0']) | first | replace('_', ':') }}"
+ansible_phydev: "ansible_{{ ansible_interfaces | sort | select('in', ['bond0_0', 'eth0']) | first }}"
```

Comment:
We should think about how to get rid of this single purpose `ansible_phydev` variable.

Very good for now! Thanks.